sql-db/panache-flyway/src/test/resources/Dockerfile

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S podman build . --tag quay.io/quarkusqeteam/wait:0.0.2 --file
+#!/usr/bin/env -S podman build . --tag quay.io/quarkusqeteam/wait:0.0.4 --file
```

Comment:
Let's remove this line altogether. Users should use [this script](https://github.com/quarkus-qe/quarkus-test-suite/blob/ca7276b430f112f9730147c6da95cd0008106c67/sql-db/panache-flyway/src/test/resources/build.sh) and not trying to execute the Docker file